_episodes/02-starting-with-data.md

```diff
@@ -18,7 +18,20 @@ objectives:
     - "Ejecutar operaciones matemáticas básicas y calcular estadísticas descriptivas sobre datos dentro de un Pandas **Dataframe**."
     - "Crear gráficos simples."
 keypoints:
-    - "FIXME"
+    - "Las bibliotecas en Python son conjuntos de herramientas, funciones y
+    estructuras de datos, agrupadas de forma conveniente para facilitar  nuestro trabajo".
+    - "Pandas es una librería que ofrece funcionalidades avanzadas para manejar
+    datos tabulares, procesarlos y obtener resultados similares a los que ofrecen
+    las hojas de cálculo."
+    - "El DataFrame es la estructura de datos fundamental de Pandas, representa una
+    tabla de datos panel con indexación integrada. Cada columna contiene los valores
+    de una variable y cada fila un conjunto de valores de cada columna."
```

Comment:
"cada fila un conjunto de valores de cada columna."
 
I found this very confusing, can we rephrase please?